src/app/(BottomTabNavigation)/Profile/DeleteAccount/styles.ts

```diff
@@ -6,13 +6,64 @@ export default StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: colors.white,
-    alignItems: 'center',
+    paddingHorizontal: 25,
+  },
+  headerContainer: {
+    alignSelf: 'flex-start',
+    marginTop: 63, // copied this value from profile screen
+  },
+  textContainer: {
+    rowGap: 20,
+    marginTop: 100,
+  },
+  topText: {
+    fontSize: 38,
+    fontStyle: 'normal',
+    fontWeight: '800',
+    color: colors.black,
+  },
+  blurb: {
+    fontSize: 16,
+    fontStyle: 'normal',
+    fontWeight: '300',
+    color: colors.black,
+  },
+  buttonContainer: {
+    flexDirection: 'row',
+    marginTop: 365, // im guessing here how do i move the buttons to the bottom of the screen w out just adding a bunch of marginTop for this container
```

Comment:
I'd recommend putting all content in a container that spans the whole screen. Then use the "space-between" option of justify-content/align-items (i forget which one).